raku/rakudo-star-support.md

```diff
@@ -0,0 +1,33 @@
+# Automated Rakudo Star Builds
+
+## Description
+
+Rakudo Star is currently build by hand on every release. This, in conjuction
+with a maintainer that doesn't use every OS in existence, causes targets other
+than GNU+Linux to be left behind. By leveraging automated builds, whether it be
+through GitHub, GitLab, SourceHut or anything else, support for other systems
+can be improved and kept up to date much easier.
+
```

Comment:
Please add here a few milestones the student would be expected to reach; one or two will be enough.